v2/stack_analyses.yaml

```diff
@@ -138,33 +143,42 @@ components:
         severity:
           type: string
           enum:
-          - low
-          - medium
-          - high
-          - critical
+            - low
+            - medium
+            - high
+            - critical
         title:
           type: string
           example: SQL Injection
         id:
           type: string
+          example: SNYK-PYTHON-DJANGO-40027
         url:
-          type: string        
+          type: string
+          example: https://snyk.io/vuln/SNYK-PYTHON-DJANGO-4002
       description: Limited private vulnerability details for free tier users
     PremiumVulnerabilityFields:
-      title: ExtendedVulnerabilityFields
+      title: PremiumVulnerabilityFields
       type: object
       properties:
         malicious:
           type: boolean
           default: false
         patch_exists:
           type: boolean
+          default: false
         fixable:
           type: boolean
+          default: false
         exploit:
           type: string
+          example: 
+            - High
+            - Not Defined
         description:
           type: string
+          example: |
+            "## Overview\r\n[`django`](https://pypi.python.org/pypi/django) is a high-level Python Web framework that encourages rapid development and clean, pragmatic design.\r\n\r\nAffected versions of this package expose sensitive information due to not properly restricting the use of a query string that performs certain object filtering. An attacker may obtain sensitive information via a series of requests containing regular expressions, as demonstrated by a `created_by__password__regex` parameter.\r\n\r\n## References\r\n- [Django Vulnerability Description](http://www.djangoproject.com/weblog/2010/dec/22/security/)\r\n- [GitHub Commit](https://github.com/django/django/commit/732198ed5c)\r\n- [Redhat Bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=665373)\r\n- [Openwall](http://www.openwall.com/lists/oss-security/2011/01/03/5)\r\n- [CVE](https://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2010-4534)\r\n"
         fixed_in:
```

Comment:
the fixed_in field is a list of string. It wont be just a string.

Reply:
example: "initiallyFixedIn": [
        "4.4.5",
        "4.5.2"
      ]

Reply:
If the values to be shown in fixed in field is our computed data, then ignore the comment as we will always show a single recommended version

Reply:
Nope, it carries snyk's info, I will fix it.